src/v2/view-builder/views/ov/EnrollOVSwitchChannelView.js

```diff
@@ -0,0 +1,28 @@
+import { loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+  title: loc('oie.enroll.okta_verify.select.channel.title', 'login'),
+  getUISchema () {
+    const schemas = BaseForm.prototype.getUISchema.apply(this, arguments);
+    const description = {
+      View: loc('oie.enroll.okta_verify.select.channel.description', 'login'),
+      selector: '.o-form-fieldset-container',
+    };
+    schemas.forEach((schema) => {
```

Comment:
This filtering might not be needed in future based on UX and some api decisions.